services/cloud_watch.rb

```diff
@@ -42,17 +40,23 @@ def receive_logs
         setting.to_s.empty?
     end
 
-    cloudwatch = AWS::CloudWatch::Client.new(
+    cloudwatch = Aws::CloudWatch::Client.new(
       region: settings[:aws_region],
-      access_key_id: settings[:aws_access_key_id],
-      secret_access_key: settings[:aws_secret_access_key],
+      credentials: Aws::Credentials.new(settings[:aws_access_key_id], settings[:aws_secret_access_key], settings[:aws_session_token]),
```

Comment:
```suggestion
      credentials: Aws::Credentials.new(settings[:aws_access_key_id], settings[:aws_secret_access_key]),
```

We won't have a session token